iso/create-osrelease.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+VERSION=`cat ../VERSION`
```

Comment:
Since this script is called from the Makefile so Version shouldn't be part  as variable here.